website/docs/Pipelines/reblock_gvcf_spec.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 14
+---
+
```

Comment:
Where will this appear in the WARP doc site navigation on the side bar? Perhaps a sub page of the Whole Genome and Exome sections? Or we could group those two under Germline and have it be a top level page there?